akka-http-core/src/main/java/akka/http/javadsl/model/Query.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (C) 2009-2014 Typesafe Inc. <http://www.typesafe.com>
+ */
+
+package akka.http.javadsl.model;
+
+import akka.http.impl.model.JavaQuery;
+import akka.http.scaladsl.model.*;
+import akka.japi.Option;
+import akka.japi.Pair;
+import akka.parboiled2.CharPredicate;
+import akka.parboiled2.ParserInput$;
+
+import java.nio.charset.Charset;
+import java.util.List;
+import java.util.Map;
+
+public abstract class Query {
```

Comment:
I just documented here as `JavaQuery` is INTERNAL and the comments would be redundant.